dev/src/Snippet/Coverage/Coverage.php

```diff
@@ -17,8 +17,14 @@
 
 namespace Google\Cloud\Dev\Snippet\Coverage;
 
+use Google\Cloud\Dev\CheckGrpcTrait;
```

Comment:
Good point, removed the trait and moved `shouldSkipGrpcTests` to `tests/GrpcTestTrait.php`